concoction-lib/src/test/java/info/mmpa/concoction/scan/BuilderTests.java

```diff
@@ -0,0 +1,44 @@
+package info.mmpa.concoction.scan;
+
+import example.RuntimeExec;
+import info.mmpa.concoction.Concoction;
+import info.mmpa.concoction.model.path.MethodPathElement;
+import info.mmpa.concoction.output.Detection;
+import info.mmpa.concoction.output.Results;
+import info.mmpa.concoction.util.TestUtils;
+import org.junit.jupiter.api.Test;
+import software.coley.collections.Sets;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.NavigableSet;
+import java.util.Set;
+
+import static info.mmpa.concoction.util.Unchecked.cast;
+import static org.junit.jupiter.api.Assertions.*;
+public class BuilderTests {
+    @Test
+    void test() {
```

Comment:
Generally tests should hone in on what the logic being tested is designed for. The builder assists in loading the input and scanning models.

While copying from the `InstructionMatchingTests` and changing the setup does acheive validation the builder works, its doing a bit extra work not related to the code we want to test here.